TASVideos/Pages/RssFeeds/Submissions.cshtml

```diff
@@ -11,7 +11,7 @@
 <rss>
 	<channel>
 		<partial model="@((title, link))" name="_RssChannel" />
-		<atom-link href="@(Settings.BaseUrl)/submissions.rss" />
+		<atom-link href="@(Settings.BaseUrl)/Submissions.rss" />
```

Comment:
Is this needed for the rss spec?  If not we dont' want to do this.  Integrating systems can be case sensitive on urls.  It was lowercase to avoid potentially something

Reply:
All our links to the rss feeds are capitalized like this, and rss requires it to be exactly the same.

Reply:
Or rather, we have some that are uppercase ( https://tasvideos.org/XMLDataFeed or https://tasvideos.org/NewMovies because they are normalized to uppercase by our wiki engine) and some that are lowercase (the link tags in our main html).
But unfortunately both capitalizations return the same rss, and this atom link needs to be exact.

Reply:
Based on the wiki and the rest of our code, it seems the original intent was for the rss urls to be uncapitalized, so `submissions.rss`.
I would suggest reverting the canonical path changes.

Reply:
I thought ASP.NET forced all its pages to be PascalCase?

Reply:
The actual page is `/RssFeeds/Submissions` but we reroute it to `/submissions.rss`.
(See `ServiceCollectionExtensions.Aliases` for all our route changes.)